rero_ils/modules/items/api/circulation.py

```diff
@@ -551,10 +551,11 @@ def checks_before_a_cancel_item_request(self, loan, **kwargs):
             LoanState.ITEM_AT_DESK in loans_list or
             LoanState.ITEM_ON_LOAN in loans_list or
             LoanState.ITEM_IN_TRANSIT_FOR_PICKUP in loans_list or
-            LoanState.ITEM_IN_TRANSIT_TO_HOUSE in loans_list
+            LoanState.ITEM_IN_TRANSIT_TO_HOUSE in loans_list or
+            LoanState.PENDING in loans_list
```

Comment:
I think that you could use this idiom :  

`any(element in list1  for element in list2)`

and do something like :
```


elif loan['state'] == LoanState.PENDING and \
    any(state in loans_list for state in [
     LoanState.ITEM_AT_DESK,
     LoanState.ITEM_ON_LOAN,
     LoanState.ITEM_IN_TRANSIT_FOR_PICKUP,
     LoanState.ITEM_IN_TRANSIT_TO_HOUSE,
     LoanState.PENDING]):
```

Reply:
Done